src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -64,24 +65,10 @@ def _get_changelog_draft_entries(
     return towncrier_output
 
 
-@lru_cache(maxsize=1, typed=True)
-def _autodetect_scm_version():
-    """Retrieve an SCM-based project version."""
-    for scm_checkout_path in Path(__file__).parents:  # noqa: WPS500
-        is_scm_checkout = (
-            (scm_checkout_path / '.git').exists() or
-            (scm_checkout_path / '.hg').exists()
-        )
-        if is_scm_checkout:
-            return get_version(root=scm_checkout_path)
-    else:
-        raise LookupError("Failed to locate the project's SCM repo")
-
-
 @lru_cache(maxsize=1, typed=True)
 def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     """Generate a fallback version string for towncrier draft."""
-    known_strategies = {'scm-draft', 'scm', 'draft', 'sphinx-version', 'sphinx-release'}
+    known_strategies = {'scm-draft', 'draft', 'sphinx-version', 'sphinx-release'}
```

Comment:
Did you miss `scm-draft`?
```suggestion
    known_strategies = {'draft', 'sphinx-version', 'sphinx-release'}
```